### DISCUSSION TITLE
Command palette opens up when pressing "Shift" or "Ctrl" on its own

### DISCUSSION THREAD

**Discussion by cheap-glitch** [community/community#13399] [Other Feature Feedback, Questions, & Ideas]
A couple of weeks ago, the Command Palette suddenly started opening when pressing <kbd>Shift</kbd> or <kbd>Ctrl</kbd> _on its own_. This makes GitHub basically unusable with the palette enabled, since the pop-up appears whenever I try using a browser shortcut, an "in-page" shortcut, or just inputting an uppercase letter.

- OS: **Arch Linux**
- Browser: **Firefox 98**
- Keyboard layout: **French (Azerty)**

-----

Side note: the implementation of keyboard shortcuts in general is basically broken for non-Qwerty users, e.g. pressing <kbd>?</kbd> (i.e. <kbd>Shift</kbd>+<kbd>,</kbd>) to open the help modal doesn't work on an Azerty layout.

**Comment by dschwilk** [comment]
Not an answer but a confirmation and elaboration.

I can confirm. This occurs in any github text entry box such as found in issues, searches, etc. Disabling the 'command mode' in the 'settings ... accessibility' menu has no effect. This means mouse context menu cut and paste from a text editor is the only way for me to get text into a github issue such as this one. 

- OS Kubuntu Linux
- Browser Firefox 98.0.2
- Keyboard layout English US

**Comment by akinley** [comment]
I’m not seeing it open with _just_ Shift, but I _am_ seeing with Shift+Win, which makes it pop up when I use OS-level keyboard shortcuts to move the browser window between monitors. Note that I’d never seen the Command Palette feature at all until today, so I guess there was some A/B testing or a phased rollout going on. Disabling the Command Palette from the Feature Preview popup does work for me.

- Linux Mint 19.3 Cinnamon
- Firefox Developer Edition 100.0 beta 1
- US English keyboard layout

**Comment by allanwalpy** [comment]
Don't have such behavior with `Ctrl` or `Shift`, but on `Win` pressing, including combination with `Win`, command pallete appears in searh mode. On disabling feature nothing appears.

- Ubuntu 20.04 Xfce
- Chrome 100.0.4896.127 
- English US keyboard layout

Command Pallete: command mode config: `control` + `shift` + `p`; search mode config: `control` + `shift` + `k`
